zkp/js/test/anon_nullifier.js

```diff
@@ -14,23 +14,18 @@
 // See the License for the specific language governing permissions and
```

Comment:
the format of this file is not consistent with other files. It should be auto formatted by prettier when running the test using `npm t`